src/rust/engine/fs/store/src/tests.rs

```diff
@@ -30,62 +28,12 @@ use crate::{
 
 pub(crate) const STORE_BATCH_API_SIZE_LIMIT: usize = 4 * 1024 * 1024;
 
-pub fn big_file_fingerprint() -> Fingerprint {
```

Comment:
The functions here have moved to `testutil` so they can be used by the `remote_provider_...` tests without duplication.